### PR TITLE
Add ability to use sprig functions as part of a template

### LINKED ISSUE #229: Include sprig functions as part of the template cache loading
In order to support more complex header and footer templates we will utilize sprig functions. Maybe more in the future but at least this for now.

### PR DESCRIPTION
**Pull request by rocktavious**
Closes #229

**Check the following**
- [ ] Keep 100% code coverage
- [ ] Be properly formatted
- [ ] Have no linting errors
- [ ] Documentation changes are included
- [ ] Include a change file if expected

### CODE REVIEW

File: .changes/unreleased/Added-20220213-200122.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+kind: Added
+body: allow for use of sprig functions in templates
```
Comment (miniscruff) [inline .changes/unreleased/Added-20220213-200122.yaml:2]:
```suggestion
body: Allow for use of sprig functions in templates
```
I have been using capitals to start change fragments

File: core/templatecache.go [hunk 2]
```diff
@@ -21,7 +23,7 @@ func (tc *TemplateCache) Load(text string) (*template.Template, error) {
 		return cachedTemplate, nil
 	}
 
-	templ, err := template.New(text).Parse(text)
+	templ, err := template.New(text).Funcs(sprig.TxtFuncMap()).Parse(text)
```
Comment (miniscruff) [inline core/templatecache.go:26]: Mind adding a test for a simple sprig function in a template, just as a sanity check.

**Review by miniscruff — CHANGES_REQUESTED** [review]

File: .changes/unreleased/Added-20220213-200122.yaml (hunk 1 above)
Comment (rocktavious) [reply .changes/unreleased/Added-20220213-200122.yaml]: Fixed

File: core/templatecache.go (hunk 2 above)
Comment (rocktavious) [reply core/templatecache.go:26]: Added

**Review by miniscruff — APPROVED** [review]

**Pull request merged**